main_test.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestAPIHostParsing(t *testing.T) {
+	u, err := parseAPIHost("api.honeycomb.io")
+	if err != nil {
+		t.Error(err)
+	}
+	if u.Scheme != "https" {
+		t.Errorf("Expected https, got: %s", u.Scheme)
+	}
```

Comment:
Would it be appropriate to add another test case for when a scheme is provided in the URL to parse?

```go
	u, err := parseAPIHost("http://some.onprem.test.host.without.a.cert.example.com"")
	if err != nil {
		t.Error(err)
	}
	if u.Scheme != "http" {
		t.Errorf("Expected http, got: %s", u.Scheme)
	}
```

Reply:
I dig it. Added!